client/platform/desktop/backend/ipcService.ts

```diff
@@ -70,6 +70,12 @@ export default function register() {
     return defaults;
   });
 
+  // eslint-disable-next-line arrow-body-style
```

Comment:
Is this eslint-dsiable needed?  A `yarn lint` isn't showing me anything and I don't think that is a rule I've ever seen enforced in any of my airbnb linting based projects.